Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -22,78 +22,131 @@ variable (F K L : Type _) [Field F] [Field K] [Field L] [Algebra F K] [Algebra F
   [IsScalarTower F K L]
 
 /-- The normal closure of `K` in `L`. -/
-noncomputable def normalClosure : IntermediateField K L :=
```

Comment:
What's the replacement if I want `IntermediateField K L` after all?

Reply:
You can get it from the `IsScalarTower` instance (turn the `IsScalarTower` into an `algHom`, and take the `fieldRange`).

Reply:
Can you add a lemma that states that `(construction_you_describe).restrictScalars K = normalClosure` (or if it's already there, can you tell me its name)?

Reply:
Done: https://github.com/leanprover-community/mathlib4/pull/6163/commits/1ac7f5eb3905a3568712aecb341d03fe5bdd715c